papers/ring_vrf/overview.tex

```diff
@@ -26,141 +34,151 @@ \section{Protocol overview}
+
+
+$$ \Setst{ ((\sk, \pk),(\Out, \msg, \ring))}{
     \eprint{
-        (\pk,\sk) \leftarrow \KeyGen, \quad %   \textrm{ and }
+        (\pk,\sk) \leftarrow \rVRF.\KeyGen(\secparam), \quad %   \textrm{ and }
         \pk \in \ring, \quad
-        \Out = \PRF(\sk,\msg)
+        \Out = \rVRF.\Eval(\sk,\msg)
     }{
         \begin{aligned}
-        & (\pk,\sk) \leftarrow \KeyGen \\
+        & (\pk,\sk) \leftarrow \KeyGen(\secparam) \\
         & \pk \in \ring \\
-        & \Out = \PRF(\sk,\msg)
+        & \Out = \rVRF.\Eval(\sk,\msg)
         \end{aligned}
     }
 } $$
-% Although convenient for security arguments, % formalization
-
-We would need time $O(|\ring|)$ in \rSign and \rVerify merely to read
-this \ring argument.  Instead, ring signatures run asymptotically faster
-by replacing the \ring argument with some set commitment to \ring,
-roughly like what ZCash does.
-\begin{itemize}
-% \item $\rVRF.\CheckRing : \ring \mapsto \comring$ takes a set \ring of public keys and returns a public key set commitment \comring.
-\item $\rVRF.\CommitRing : (\ring,\pk) \mapsto (\comring,\openring)$ \,
-    returns a commitment for a set \ring of public keys, and
-    optionally the operning \openring for some $\pk \in \ring$ as well.
-\item $\rVRF.\OpenRing : (\comring,\openring) \mapsto \pk \, \lor \perp$ \,
-    returns a public key \pk, provided \openring correctly opens
-    the ring commitment \comring, or failure $\perp$ otherwise.
-\end{itemize}
-
-We thus replace the membership condition $\pk \in \ring$ in the above
-language and NIZK by the by the opening condition
-$$ \exists \openring \textrm{\ s.t.\ } \pk = \OpenRing(\comring,\openring) \mathperiod $$
-% $\pk = \OpenRing(\comring,\openring)$.
-%
-% $$ \pi_0 = \NIZK \Setst{ \Out, \msg, \comring }{
-%     \begin{aligned}
-%         (\pk,\sk) &\leftarrow \KeyGen,  \quad
-%           \Out = \PRF(\sk,\msg)  \\
-%         \exists \openring \textrm{\ s.t.\ }
-%           \pk &= \OpenRing(\comring,\openring)  \\      
-%     \end{aligned}
-% } $$
-
 
-Aside from proving a PRF evaluation, an EC VRF should typically sign
-some auxiliary data \aux by hashing \aux when computing its challenge,
-but doing so only saves one redundant signature verification for EC VRF.
-Auxiliary data becomes critical for ring VRFs, because otherwise
-unmoored ring VRFs invariably permit replay attacks.
-As an example, if ring VRFs authorized creating blocks in an anonymous
-praos \cite{praos} blockchain then \aux must include the block being
-created, or else others could steal the block production turn.%
-\footnote{We suppress multiple input-output pairs until \S\ref{subsec:multi_io} below, but they work like in \cite{PrivacyPass}.}
+So, while a verifier verifies a signature $ \sigma $ of a message $ \msg $ with a set of public key $ \ring $, she should be convinced that $ \Out $ is generated for the input $ \msg $ with a secret key $ \sk $ of which the  public key $ \pk $ is in the set  $ \ring $. 
```

Comment:
This is redundant to the NIZK, but a shorter version before the NIZK is fine.  Adding.

Reply:
Added something vaguely based upon the next part too.